src/webview/MessageList.js

```diff
@@ -113,45 +115,33 @@ export type Props = $ReadOnly<{|
 |}>;
 
 /**
- * A URL-like magic string denoting the webview-assets folder.
+ * The URL of the platform-specific assets folder.
  *
- * Not suitable for reuse or export.
+ * - On iOS: We can't easily hardcode this because it includes UUIDs.
+ *   So we bring it over the React Native bridge in ZLPConstants.m.
+ * - On Android: Different apps' WebViews see different (virtual) root
+ *   directories as `file:///`, and in particular the WebView provides
+ *   the APK's `assets/` directory as `file:///android_asset/`. [1]
+ *   We can easily hardcode that, so we do.
+ *
+ * [1] Oddly, this essential feature doesn't seem to be documented!  It's
+ *     widely described in how-tos across the web and StackOverflow answers.
+ *     It's assumed in some related docs which mention it in passing, and
+ *     treated matter-of-factly in some Chromium bug threads.  Details at:
+ *     https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/android.20filesystem/near/796440
+ */
+const assetsUrl =
+  Platform.OS === 'ios'
+    ? new WhatwgURL(NativeModules.ZLPConstants.resourceURL)
+    : new WhatwgURL('file:///android_asset/');
+
+/**
+ * The URL of the webview-assets folder.
+ *
+ * See `check_path_name` in `tools/build-webview` for information on
+ * what folder this is.
```

Comment:
This is implementation details, not interface, so doesn't belong in jsdoc.

Reply:
And thinking a bit more about this, the real value of this cross-reference is about finding what files go into this directory, not finding where it lives in the build tree. (The existing comment isn't very specific about this distinction.) So instead of mentioning `check_path_name`, maybe:
```suggestion
 *
 * This is the folder populated at build time by `tools/build-webview`.
```
(and jsdoc is a fine place for that.)